### PR TITLE
Adding support to parked interactions

### PR DESCRIPTION
**Pull request by grobira**
Hi @guvarallo I've used your plugin in a project and noticed that it doesn't support sending messages to address that are already with an opened conversation/interaction. 
I've just added a couple if to check if the address already have an opened conversation.

I've also added a sequence diagram that explains the solution.
Outbound Whatsapp

### CODE REVIEW

File: plugin-outbound-whatsapp/src/utils/templates.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
 export const templates = [
-  'Thank you for purchasing Saul Goodman! We value your feedback and would like to learn more about your experience.',
-  'Hi Saul Goodman, were we able to solve the issue that you were facing?'
+  'Hi customer, were we able to solve the issue that you were facing?'
```
Comment (guvarallo) [inline plugin-outbound-whatsapp/src/utils/templates.js]: No need to change this for now, we will remove this file soon to add a field to upload templates

File: plugin-outbound-whatsapp/.env.example [hunk 2]
```diff
@@ -1,6 +0,0 @@
-FLEX_APP_TWILIO_SERVERLESS_DOMAIN=
```
Comment (guvarallo) [inline plugin-outbound-whatsapp/.env.example:1]: We need this file, cannot be removed

File: serverless-outbound-whatsapp/functions/sendOutboundWA.js [hunk 3]
```diff
@@ -52,14 +52,79 @@ const openAChatTask = async (
   }
 }
 
+const fetchPreviousConversations = async (client, To) => {
+
+  // const resUserConversations = await axios.get(`https://conversations.twilio.com/v1/ParticipantConversations?Address=whatsapp:${To}`, {
```
Comment (guvarallo) [inline serverless-outbound-whatsapp/functions/sendOutboundWA.js]: Remove comments here

**Review by guvarallo — CHANGES_REQUESTED** [review]
Really nice job Giovanni, thanks for this, loved the updates for existing interactions. Left just a few comments and we're good to go.

**Comment by grobira** [comment]
@guvarallo Feito!

**Pull request merged**